src/signals/incident-management/components/FilterForm/FilterForm.tsx

```diff
@@ -311,9 +310,9 @@ const FilterForm = ({
     [dispatch]
   )
 
-  const onAddressBlur = useCallback(
+  const onAddressSelect = useCallback(
     (event) => {
```

Comment:
Not really an event anymore I think. It's just data passed through.

Reply:
Vrrrm, it just passed through. :grin: